src/components/Root/Root.tsx

```diff
@@ -8,27 +8,24 @@ import { IS_DEMO_MODE, IS_DEVELOPMENT_BUILD, IS_PREVIEW_MODE } from '#src/utils/
 import DemoConfigDialog from '#components/DemoConfigDialog/DemoConfigDialog';
 import LoadingOverlay from '#components/LoadingOverlay/LoadingOverlay';
 import DevConfigSelector from '#components/DevConfigSelector/DevConfigSelector';
-import { useConfigSource } from '#src/utils/configOverride';
-import { loadAndValidateConfig } from '#src/utils/configLoad';
-import { initSettings } from '#src/stores/SettingsController';
 import AppRoutes from '#src/containers/AppRoutes/AppRoutes';
 import registerCustomScreens from '#src/screenMapping';
+import { initApp } from '#src/modules/init';
+import { useConfigSource } from '#src/utils/configOverride';
+import { useFeaturesStore } from '#src/stores/FeaturesStore';
 
 // This is moved to a separate, parallel component to reduce rerenders
 const RootLoader = ({ setAppIsReady }: { setAppIsReady: React.Dispatch<React.SetStateAction<boolean>> }) => {
   const { t } = useTranslation('error');
-  const settingsQuery = useQuery('settings-init', initSettings, {
```

Comment:
There are a couple other comments in the PR around this, but this is the most direct place to ask, Why did you remove this?

Reply:
I moved it to the build time compilation to make the code cleaner and more structured. That way we have:

1. One entry point to initialize the whole application (we don't split the logic into settings init and config init)
2. Clear structure where all controllers and services are classes. We won't have to make a settings controller a function. In case we make it a class, then we will have to split container initialization and do it the way I proposed in the very first version.
3. We won't have to trigger this 'endpoint' and read the file every time a customer loads the page.
4. We don't need to support an additional store which is basically a configuration file.

I understand your point that customers can modify this configuration and replace file when needed with no rebuild procedure. Do we know whether they use this feature? Or do they use it just once when they download a .zip archive where they could replace the file?

Reply:
Yes, those are the 2 main use cases. Supporting customers who may not have engineering resources to compile from source. Idk if it's widely used, but I think it's nice to maintain the capability.

Reply:
Loading the ini file is really no different than loading any other resource like a js file or image. I don't think it will be a big bottleneck. It has to be split from config init for the reason that the config init depends on the values in the settings.

If we want to call the settings loader something besides a controller just because it's a function export, that's fine with me. I don't remember why I called it a controller in the first place. Or you could also make it a class with a single init method if you want. The difficulty with the early PR iirc had to do with trying to resolve the settings or config controller using inversify, which doesn't really need to happen regardless because there's no alternative module for those.

As for using 1 more store, I don't think that's such a big deal. If you want we could combine it into the config store, but for that matter we could just have 1 store for all of the stored values.